docs/TypeScript.md

```diff
@@ -126,14 +127,80 @@ The type system heavily relies on generic properties to provide the most accurat
 
 ### JSON Schema
 
+To validate your requests and responses you can use JSON Schema files. If you didn't know already, defining schemas for your Fastify routes can increase their throughput! Check out the [Validation and Serialization](Validation-and-Serialization.md) documentation for more info.
+
+Also it has the advantage to use the defined type within your handlers (including pre-validation, etc.).
+
+Here are some options how to achieve this.
+
+
+#### Typebuilder
```

Comment:
This paragraph talks about `typebox` but it is named `Typebuilder`, why? Let's rename to typebox!

```suggestion
#### typebox
```